docs/platforms/rust/common/logs/index.mdx

```diff
@@ -70,7 +70,7 @@ As of today, array and object types will be converted to strings using their JSO
 
 ## Integrations
 
-### `tracing` Integration
+### `tokio-rs/tracing` Integration
 
 To use the `tracing` integration with logs, add the necessary dependencies to your `Cargo.toml`.
```

Comment:
m: Assuming we keep the name change above, please also use the same name consistently throughout the doc

```suggestion
To use the `tokio-rs/tracing` integration with logs, add the necessary dependencies to your `Cargo.toml`.
```